oeps/oep-0002.rst

```diff
@@ -69,6 +70,39 @@ following fields:
     If an OEP isn't listed in the ``oeps`` dictionary, then it is assumed to be
     ``False``, unless the reporting tool can auto-detect accordance.
 
+``track-pulls``: boolean (optional)
+
+    .. This should be moved to a separate OEP
+
+    Whether or not this repository should be tracked for creation of OSPR tickets.
+
+``openedx-release``: dictionary (optional)
+
+    .. This should be moved to a separate OEP
+
+    This controls how this repo is managed during the Open edX release process.
+
+    Possible keys:
+
+    ``ref``
+        The name of a branch in this repo. This is the branch that will be tagged
+        when an Open edX release is made.
+
+    ``requirements``
+        The path to a requirements.txt-style file in this repo. This file will
+        be consulted to find the commit to tag in dependent repos indicated by
+        ``parent-repo`` keys.
+
+    ``parent-repo``
+        The name of another repo, one that installs this repo via a requirements
+        file.  The parent repo's requirements file will indicate the commit to
+        tag in this repo.
+
+``dead``: boolean (optional)
```

Comment:
Why can't we use a more standard (and less morbid) term like `obsolete` here? (You already use that in the description.)